src/django/api/models.py

```diff
@@ -740,3 +740,37 @@ class Meta:
     def __str__(self):
         return '{0} - {1} - {2}'.format(self.facility_list_item, self.facility,
                                         self.status)
+
+
+class FaclilityAlias(models.Model):
```

Comment:
I set this up and it worked except that when I ran this in `shell_plus` it reported an error.

```
>>> FacilityAlias.objects.all()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
NameError: name 'FacilityAlias' is not defined
```

Turns out there's a typo in the name of this model: `FaclilityAlias` -> `FacilityAlias`

Reply:
Sorry for not giving this a more thorough test. Will fix.

Reply:
I fixed the typo in 974d8da.

Reply:
No worries! Checked out the newer commits and everything's working as expected now.